d8content/metadatadisplays/iiif_manifest_2.1.twig.txt

```diff
@@ -44,7 +44,7 @@ Updated Dec 14th 2020
 {
 "label": "description",
 "value": {{ data.description|json_encode|raw }}
-},
+},pgrade from 19
```

Comment:
@giancarlobi this seems to be a typo

Reply:
@DiegoPino Sorry friend, of course, It is, it has to be the same than here https://github.com/esmero/archipelago-deployment/blob/1.0.0-RC1D9/d8content/metadatadisplays/iiif_manifest_2.1.twig.txt. Do I have to change, right? I don't know the better workflow for this, tks

Reply:
Yes. Edit the file directly but on this branch
https://github.com/esmero/archipelago-deployment/tree/giancarlobi-patch-pdfinfo
Then the pull will be updated automatically

Reply:
Done, I hope in the right way, friend